tokio/src/task/scope/cancellation_token.rs

```diff
@@ -0,0 +1,255 @@
+//! An asynchronously awaitable event for signalization between tasks
```

Comment:
I don't think "signalization" is a word :)

Reply:
For non native speakers it works fine 😂
I first wanted to say "please provide a recommendation for an update" - but it's wrong anyway - the description was copied from futures-intrusive `ManualResetEvent`, but this is a `CancellationToken`